build.sbt

```diff
@@ -44,6 +46,40 @@ lazy val toolkitTest = crossProject(JVMPlatform, JSPlatform, NativePlatform)
     mimaPreviousArtifacts := Set()
   )
 
+lazy val toolkitTesting = project
+  .in(file("toolkit-testing"))
+  .settings(
+    name := "toolkit-testing",
+    Test / test := (Test / test).dependsOn(toolkit.jvm / publishLocal).value,
```

Comment:
let's make this a `crossProject(JVM, JS, Native)`. Then in `{jvm,js,native}Settings` we can do the appropriate:
```scala
Test / test := (Test / test).dependsOn(toolkit.{jvm,js,native}/ publishLocal).value
```

Reply:
hmm, and how can we pull the JVM dep of scala-cli? should we declare it w/o the `%%%` ?

Reply:
> should we declare it w/o the `%%%` ?

Yeah it's cheaty but would totally work. The other option is we put the dependency in its own project somewhere in the build instead of on this project, and just use that project as a way to generate a useful classpath.